inst/templates/methods-nab/nab-statistical-methods.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+title: "Statistical Methods"
+output: github_document
+---
+
+<!-- This file was adapted from VISCtemplates {{pkg_ver}}. -->
+
+## Statistical Endpoints
+
+<!-- Animal studies usually use SLA-MLV-subtracted values. Check the lab study plan
+for more info. -->
+
+Response to a pseudovirus was considered to be positive if the net ID~50~ titer 
```

Comment:
I think it can be kept as one template but we could put all references to [net] in brackets.

Reply:
I added a note at the top:

> If using SLA-MLV-subtracted values, replace "ID~50~ titer" with "net ID~50~ titer" or "MLV-subtracted titer" (use one, be consistent).

We want to avoid two templates, that's more to maintain and will become a source of conflicting updates.